jobserver/templates/job_detail_tw.html

```diff
@@ -129,7 +128,14 @@ <h1 class="text-3xl break-all md:bread-words md:text-4xl font-bold text-slate-90
           {% if job.action in job.job_request.cancelled_actions %}
             <p>A User has requested this Job is cancelled.</p>
           {% endif %}
-          {% status_hint job %}
+          {% if log_path %}
```

Comment:
What if the job exited with an error on a non-TPP backend, wouldn't we still want to show an error message here, just without the log path?

Reply:
No, this error (according to the old template tag) is only specific to TPP.  I'm just copying the existing logic over to unstick Tom 🙂